src/main/resources/ValidationMessages.properties

```diff
@@ -1,4 +1,7 @@
 enterDetails.companyNumber.wrongLength=Enter the company number
 
-enterDetails.penaltyNumber.emptyField=Enter the penalty reference
-enterDetails.penaltyNumber.wrongLength=Enter your penalty reference exactly as shown on your penalty letter
+enterDetails.penaltyRef.isEmpty.LATE_FILING=Enter the reference number
+enterDetails.penaltyRef.isWrong.LATE_FILING=Enter your reference number exactly as shown on your penalty letter
+
+enterDetails.penaltyRef.isEmpty.SANCTIONS=Enter the penalty reference
+enterDetails.penaltyRef.isWrong.SANCTIONS=Enter your penalty reference exactly as shown on your penalty letter
```

Comment:
Same here

Reply:
Fixed in commit [54fb8a1](https://github.com/companieshouse/penalty-payment-web/pull/180/commits/54fb8a152d4f8c1ee331f247866d23cabdf57278). I've updated the message key names to be `notEmpty` and `notValid` in `ValidationMessages.properties` following the comments.